From e8f521154dbf16a91cba229f7514cca8e136702e Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Mon, 6 Jan 2020 11:48:42 +0200
Subject: [PATCH] arm64: dts: fsl-ls1028a: add labels to Ethernet switch ports

Labels are used to name switch port net devices in Linux, use more
convenient names to make it simpler for users.

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -806,21 +806,25 @@
 					mscc_felix_port0: port@0 {
 						reg = <0>;
 						status = "disabled";
+						label = "swp0";
 					};
 
 					mscc_felix_port1: port@1 {
 						reg = <1>;
 						status = "disabled";
+						label = "swp1";
 					};
 
 					mscc_felix_port2: port@2 {
 						reg = <2>;
 						status = "disabled";
+						label = "swp2";
 					};
 
 					mscc_felix_port3: port@3 {
 						reg = <3>;
 						status = "disabled";
+						label = "swp3";
 					};
 
 					/* internal to-cpu ports */
